content/accounts.md

```diff
@@ -0,0 +1,702 @@
+
+- [`useraccounts:unstyled`](https://atmospherejs.com/useraccounts/unstyled) which lets you bring your own CSS; this one is used in the Todos example app to make the login UI blend seamlessly with the rest of the app.
+- Pre-built templates for [Bootstrap, Semantic UI, Materialize, and more](http://useraccounts.meteor.com/). These templates don't come with the actual CSS framework, so you can pick your favorite Bootstrap package, for example.
+
+While it's optional and the basic functionality will work without it, it's also a good idea to pick a router integration:
+
+- [Flow Router](https://atmospherejs.com/useraccounts/flow-routing), the router [recommended in this guide](routing.html).
+- [Iron Router](https://atmospherejs.com/useraccounts/iron-routing), another popular router in the Meteor community.
+
+In the example app we are using the Flow Router integration with great success. Some of the later sections will cover how to customize the routes and templates to fit your app better.
+
+<h3 id="useraccounts-drop-in">Drop-in UI without routing</h3>
+
+If you don't want to configure routing for your login flow, you can just drop in a self-managing accounts screen. Wherever you want the accounts UI template to render, just include the `atForm` template, like so:
+
+```html
+{{> atForm}}
+```
+
+Once you configure routing according to the section below, you'll want to remove this inclusion.
+
+<h3 id="useraccounts-customizing-templates">Customizing templates</h3>
+
+For some apps, the off-the-shelf login templates provided by the various `useraccounts` UI packages will work as-is, but most apps will want to customize some of the presentation. There's a simple way to do that using the template replacement functionality of the `aldeed:template-extension` package.
+
+First, figure out which template you want to replace by looking at the source code of the package. For example, in the `useraccounts:unstyled` package, the templates are listed [in this directory on GitHub](https://github.com/meteor-useraccounts/unstyled/tree/master/lib). By squinting at the file names and looking for some of the HTML strings, we can figure out that we might be interested in replacing the `atPwdFormBtn` template. Let's take a look at the original template:
+
+```html
+<template name="atPwdFormBtn">
+  <button type="submit" class="at-btn submit {{submitDisabled}}" id="at-btn">
+    {{buttonText}}
+  </button>
+</template>
+```
+
+Once you've identified which template you need to replace, define a new template. In this case, we want to modify the class on the button to work with the CSS for the rest of the app. Make sure that it renders the helpers you need just like the original, for example `buttonText`:
+
```

Comment:
Yep, I definitely checked that it worked in my example app :] but it's a good pattern to follow anyway, since a form could easily have two different buttons.